examples/00-fluent/exhaust_system.py

```diff
@@ -82,6 +81,7 @@
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Import the CAD geometry file (``exhaust_system.fmd``) and selectively manage some
 # parts.
+# Execute meshing.upload(import_file_name) if Fluent is running in Ansys Lab.
```

Comment:
Is the precise condition for this and other similar scenarios not something like _if Fluent is running remotely and the file is not already available to it at the relevant path_?

The way we could structure it to make it unambiguous and easy for users is:
```Python
# Uncomment the following line of code if Fluent is running remotely and the file specified by 
# import_file_name is not available to it at the relevant path
# meshing.upload(import_file_name)
```

Reply:
@seanpearsonuk 
No, it's case when the file exists on host machine or Ansys Lab's user directory but not uploaded to the server using PIM service. 

We are already handling this on the backend for most of the cases but `Execute()` or some `PartManagement` methods do not upload required files seamlessly therefore we are adding this comment.

Reply:
We can update it as follows - 

```
# Uncomment the following line of code if Fluent is running in Ansys Lab 
# meshing.upload(import_file_name)
```

Reply:
In my original version I deliberately left out the space between `#` and `meshing`. That makes it "impossible" (within reason) to implement uncommenting incorrectly. Taken literally _uncomment the line_ means remove the comment character and leave everything else in tact, hence no space.  

I still don't understand the distinction between _Lab_ and any other client-server set-up. Let's discuss next week (scrum).

Reply:
@seanpearsonuk 

`# meshing.upload(import_file_name)` and `#meshing.upload(import_file_name)` behaves same after uncommenting, if we use `Ctrl + ?` on selected line of code.

I had considered that user will use an editor shortcut to uncomment the above line of code.

Pycharm

![image](https://github.com/ansys/pyfluent/assets/106588300/bea21c7c-6592-4bd5-a9c7-7af5bb478d5b)

VS code

![image](https://github.com/ansys/pyfluent/assets/106588300/82c1150e-0beb-4281-b28b-94da0e00f7a0)

Reply:
Sure. We will discuss above point in next scrum.

Reply:
> @seanpearsonuk
> 
> `# meshing.upload(import_file_name)` and `#meshing.upload(import_file_name)` behave same after uncommenting, if we use `Ctrl + ?` on selected line of code.
> 
> I had considered that user will use an editor shortcut to uncomment the above line of code.
> 
> Pycharm
> 
> ![image](https://private-user-images.githubusercontent.com/106588300/344193993-bea21c7c-6592-4bd5-a9c7-7af5bb478d5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1ODUzOTAsIm5iZiI6MTcxOTU4NTA5MCwicGF0aCI6Ii8xMDY1ODgzMDAvMzQ0MTkzOTkzLWJlYTIxYzdjLTY1OTItNGJkNS1hOWM3LTdhZjViYjQ3OGQ1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOFQxNDMxMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NmE4YzM1OTMxNWM0ZDAwNzRlMmNlZmM0ZTFjYTE3ZTA0ZGYzYTZmZTBkYTc3NzQ5Mzc5OGI0ZGY5NTZjMGY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._l9r6YEqvoTl8vTihaBK36PyuJndo_aH6zntYgf7PjI)
> 
> VS code
> 
> ![image](https://private-user-images.githubusercontent.com/106588300/344194808-82c1150e-0beb-4281-b28b-94da0e00f7a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1ODUzOTAsIm5iZiI6MTcxOTU4NTA5MCwicGF0aCI6Ii8xMDY1ODgzMDAvMzQ0MTk0ODA4LTgyYzExNTBlLTBiZWItNDI4MS1iMjhiLTk0ZGEwZTAwZjdhMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOFQxNDMxMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMWI0YmVlNzJkYTljMzQxNDBlOTk3NDQ0NDYyYzQ0MzJhNjYzOWVkZmZiMDkzMTExZGZiNzg2ZDY0ZjQ5MWM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.oac-sYA-QOm9TdEFxgHeYe-kDwzWY_IlMXd5DwG6TM8)

It's up to the user how he's going to do it. The point is to remove the space from the original script so that it's **definitely** not there on uncommenting. As mentioned before, uncommenting means remove only the comment character which leaves the Python invalid (`IndentationError`).